src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -796,6 +796,7 @@ partial class C
         [Fact]
         public void PartialRecord_ParametersInScopeOfBothParts()
         {
+            // PROTOTYPE(record-structs): ported
```

Comment:
Not much. Just helps keep track of which tests have been duplicated/ported for record-structs. I think this will help with feature review at the end (are there some more tests that we want to move over?).  I'll bulk remove those comments once the feature review is done.

---
In reply to: [589957946](https://github.com/dotnet/roslyn/pull/51720#discussion_r589957946) [](ancestors = 589957946)